tox.ini

```diff
@@ -1,4 +1,3 @@
 [flake8]
-python: 3
```

Comment:
I feel like we keep adding and removing this from the tox.ini. What should it be?

Reply:
It is not needed here, but is needed in the stickler yaml.